oam/install_scripts/columnstore-post-install.in

```diff
@@ -4,11 +4,33 @@
 #
 # Post-install steps for columnstore install
 
+running_systemd() {
+   if  [ "$(ps --no-headers -o comm 1)" == "systemd" ]; then
+       echo 0
+   else
+       echo 1
+   fi
+}
+
 rpmmode=install
 user=`whoami 2>/dev/null`
 
 quiet=0
 
+stop_mysqld=0
+if [ -z "$(pgrep -x mysqld)" ];then
+
+    # Startup mysqld 
+    systemctl cat mariadb.service > /dev/null 2>&1
```

Comment:
I found out that service might be masked(/etc/systemd/system/mariadb.service is a symbolic link to /dev/null) but `systemctl cat mariadb.service` returns 0. I'll fix this after we merge this one.